lib/queue-that.js

```diff
@@ -5,8 +5,8 @@ var createLocalStorageAdapter = require('./local-storage-adapter')
 var createGlobalVariableAdapter = require('./global-variable-adapter')
 
 var DEFAULT_QUEUE_LABEL = 'Queue That'
-var QUEUE_POLL_INTERVAL = 100
-var ACTIVE_QUEUE_EXPIRE_TIME = 3000
+var QUEUE_POLL_INTERVAL = 20
```

Comment:
Won't this produce very frequent localStorage reads that we'd otherwise like to avoid? I guess most of the time the queue itself will be read from the memory cache if not changed, but the active queue details will be written every time? It's small but a localStorage operation none the less.